vissl/utils/io.py

```diff
@@ -51,11 +51,13 @@ def create_file_symlink(file1, file2):
         logging.info(f"Could NOT create symlink. Error: {e}")
 
 
-def save_file(data, filename):
+def save_file(data, filename,append_to_json=False):
     """
     Common i/o utility to handle saving data to various file formats.
     Supported:
         .pkl, .pickle, .npy, .json
+    Specifcally for .json, users have the option to eithier append 
```

Comment:
nit: Specifically

Reply:
With 896a74b, I have fixed the typos and changed the default to True, as it was rewriting in a lot of cases when it should have been appending. This means that the associated tutorials need to be updated as well either to set append_to_json=False or to at least add a comment in the notebook stating that re-running the cell would throw a JSON Decoder error.